src/viam/ur/module/ur_arm_state.cpp

```diff
@@ -130,30 +131,38 @@ std::unique_ptr<URArm::state_> URArm::state_::create(std::string configured_mode
                                           std::move(csv_output_path),
                                           std::move(threshold),
                                           std::move(frequency),
-
                                           ports);
 
     state->set_speed(degrees_to_radians(find_config_attribute<double>(config, "speed_degs_per_sec").value()));
     state->set_acceleration(degrees_to_radians(find_config_attribute<double>(config, "acceleration_degs_per_sec2").value()));
 
+    // Hold the mutex while we start the worker thread. It will not be
```

Comment:
This was always true but I realized it wasn't obvious at all, so I left a comment explaining how it works.